sdk/core/core-amqp/package.json

```diff
@@ -71,6 +71,8 @@
     "rhea-promise": "^0.1.15"
   },
   "devDependencies": {
+    "@azure/identity": "^0.1.0",
+    "@azure/core-http": "^1.2.6",
```

Comment:
These are used in the actual source code and not just test or samples. Therefore, these should be under `dependencies`